reports/historical/_gmmp_2015_importer.py

```diff
@@ -1,5 +1,9 @@
 from .canon import canon
 from ._base_importer import BaseReportImporter
+from reports.report_details import get_regions
+
+
+CONTINENTS = [region.lower() for _, region in get_regions() if region.lower() != 'pacific']
```

Comment:
```suggestion
REGIONS = [region.lower() for _, region in get_regions() if region.lower() != 'pacific']
```

Why the `pacific` check? Can we add a comment there for future reference